cmd/elasticindexer/config/prefs.toml

```diff
@@ -0,0 +1,16 @@
+[config]
+    enabled-indices =  [
```

Comment:
as agreed, let's move back in the config this list, we will call it `available-indices` and in this prefs.toml we will have the `disabled-indices`